static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx

```diff
@@ -69,6 +69,17 @@ export function SdkDocumentation({
     );
   }
 
+  if (!id) {
+    return (
+      <LoadingError
+        message={t(
+          'We encountered an issue while loading Client Keys for this getting started documentation.'
```

Comment:
Users might not yet know what `Client Keys` or `DSN` are, but they won’t encounter these error messages on the happy path